server/src/main/scala/za/co/absa/atum/server/model/CirceJsonImplicits.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.model
+
+import io.circe._
+import io.circe.generic.semiauto._
+import za.co.absa.atum.model.dto.MeasureResultDTO.ResultValueType
+import za.co.absa.atum.model.dto._
+
+object CirceJsonImplicits {
```

Comment:
This object shouldn't even exist. Encoder/Decoder instances should be defined in companions of given types or in either separate objects for third party types (or in SerializationUtils) but anyway it belongs to model module.

Reply:
Okay, I am thinking that if we can have "JsonImplicits" object in the utils of the model and define all our Json implicit sterilization.